### ISSUE TITLE
improve error message when version is not found

### ISSUE THREAD

**Issue by Lutzifer** [yonaskolb/Mint#74]
```
mint run krzysztofzablocki/Sourcery@0.12                                        
🌱  Cloning https://github.com/krzysztofzablocki/Sourcery.git 0.12...
🌱  Git repo not found at "https://github.com/krzysztofzablocki/Sourcery.git"
```

Better error message would be 
```
🌱  Commitish (or branch/tag) 0.12  not found at "https://github.com/krzysztofzablocki/Sourcery.git"
```

**Comment by yonaskolb** [comment]
While the message still isn't as friendly, this is improved in #116 by providing the actual git error

**Comment by yonaskolb** [comment]
Fixed in 0.11.3
